test/testcafe/spec/ChallengeFactorEmailMagicLink_spec.js

```diff
@@ -43,20 +43,27 @@ test
     await setup(t);
     const terminalPageObject = await new TerminalPageObject(t);
     await t.expect(terminalPageObject.getMessages()).eql('Please return to the original tab.');
+    await t.expect(await terminalPageObject.goBackLinkExists()).notOk();
```

Comment:
`ChallengeFactorEmailMagicLink_spec.js` is deprecated (test with `Factor` name are legacy) . are you seeing error because of your change?

Reply:
bacon is green https://bacon-go.aue1e.saasure.net/#!/commits/okta-signin-widget/sha/2b215450a475c9060ebae4e56b0fd92ab343a7ed, but wasn't when I have bug in my change.

Reply:
I didn't follow.

Reply:
I am not sure why you asked about errors and what errors did you mean - I don't see any errors with the bacon recent build.
Please be specific about the class/place if I need to move tests to another place.

Reply:
for example this build https://bacon-go.aue1e.saasure.net/#!/commits/okta-signin-widget/sha/a4f274a5538ae2d5f8c97f8b579576125d929863 was failing because of the bugs in tests including `ChallengeFactorEmailMagicLink_spec.js`